src/ray/core_worker/core_worker_process.cc

```diff
@@ -88,9 +88,12 @@ CoreWorkerProcessImpl::CoreWorkerProcessImpl(const CoreWorkerOptions &options)
     const std::string app_name = app_name_ss.str();
     const std::string log_filepath =
         RayLog::GetLogFilepathFromDirectory(options_.log_dir, /*app_name=*/app_name);
+    const std::string err_log_filepath =
+        RayLog::GetErrLogFilepathFromDirectory(options_.log_dir, /*app_name=*/app_name);
     RayLog::StartRayLog(app_name,
                         RayLogLevel::INFO,
                         log_filepath,
+                        err_log_filepath,
```

Comment:
Actually if we remove this then it's the same old behavior? Worker C++ err logs go to stderr